### DISCUSSION TITLE
MKT Toolbox Import

### DISCUSSION THREAD

**Discussion by ghost** [MKT-Playbook/Hub#32] [Feature requests]
Hey

it it would awesome to be amble to import data also. 

At the moment, MKT Toolbox adds users to synch the account directly. It would be great to import these data optional in case one forgets to take screenshots  :-)

**Comment by TitusPosiadly** [comment]
Yes, this is also something I want to include in the next version. I know Sam is/was working on an API that could automate this, but until then I will add a simple CSV importer. This way users can export the CSV data from the "Import Wizard" in the MKT Toolbox and import it into the app via copy-paste.

**Comment by TitusPosiadly** [comment]
Hey, the CSV importer is currently being beta tested. I would love to have you as a beta tester as well (in case you aren't already). Just shoot me an email and I will send the invite to you right away.
